deployments/reviewer/libero-reviewer--prod.yaml

```diff
@@ -1,21 +1,21 @@
 ---
-apiVersion: helm.fluxcd.io/v1
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
 kind: HelmRelease
 metadata:
   name: libero-reviewer--prod
   namespace: reviewer
-  annotations:
-    fluxcd.io/automated: "false"
```

Comment:
Not sure if you spotted this, but the automation for prod was purposely disabled.

The old workflow was that as soon as the deployment to staging was successful and the tests passed then production would be updated, but I found that workflow a little too bold hence I disabled the automation for prod in favour of just manually updating the tags as and when required.

I think it might be best to still ensure this is disabled for now, but you can then review and re-enable it later if you so desire.

Reply:
As yet, the automated updates aren't turned on, and wouldn't be with this PR. The new resources so far (and in this PR) are just for flux's knowledge of what the latest image is, similar to the other annotations on the old helm release. I had in mind that I was only going to enable the staging when it came around to it (after migrating all helm releases and fully removing fluxv1), but also didn't want to lose the setup stored in those annotations, which is why I've been migrating them with the helm releases, if that makes sense?

Reply:
Yep, makes sense to me